content(hackathon): traducir info al inglés (#184)
somosnlp/somosnlp.org: pages/hackathon/cartel.en.md

---
title: "Somos NLP Hackathon 2024: Design the Poster"
description: Design the Poster for the Somos NLP Hackathon 2024!

> lang: eng

Reviewer: ```suggestion
lang: en
```

cover: https://somosnlp.github.io/assets/images/eventos/20230301_hackathon_wip.png
---

> To promote different skills in our community, we want to allow you to design the hackathon poster this year. The objectives of this year's hackathon are the creation of a quality Spanish corpus that includes the varieties of this rich language spoken by 600M people and to start a leaderboard that allows us to evaluate LLMs in Spanish. Show your most artistic version and help us create the poster!

Reviewer: ```suggestion
To promote different skills in our community, this year we want to give you the chance to design the hackathon poster. The objectives of this year's hackathon are the creation of a quality Spanish corpus that includes the varieties of this rich language spoken by 600M people and the creation of a leaderboard that allows us to evaluate LLMs in Spanish. Show your most artistic version and help us create the poster!
```

## 🎨 How to participate

1. Design a poster (o more)
2. [Send it to us](https://forms.gle/iJrKZtBXvnAk5Ji38) before January 30th
3. We will publicly thank all proposals (if you want, of course)
4. If you win, your poster will be the image of the 2024 hackathon!

## ✅ What does the poster have to include

- Dimensions: 1280 × 720 minimum with 16:9 aspect ratio
- Title: #Somos600M
- Edition: Third
- Date: March 1 to 24, 2024

> - Website: Somosnlp.org/hackathon

Reviewer: ```suggestion
- Website: somosnlp.org/hackathon
```

- #HackathonSomosNLP
- [Somos NLP Logo](https://github.com/somosnlp/assets/blob/main/logo_somos_nlp.png)

## 📝 Notes

- Creations must be original
- The use of all types of design tools is allowed, including AI systems
- It will be necessary to be transparent regarding the use of AI

## 🤔 Inspiration

> - For example, in the edition of [2023](https://somosnlp.org/hackathon-2023) we use Midjourney for the creation of the "person-book"

Reviewer: ```suggestion
- For example, in the edition of [2023](https://somosnlp.org/hackathon-2023) we used Midjourney for the creation of the "person-book"
```

- It may help you to think that, once the hackathon poster has been selected, we will have to design posters for the talks

We look forward to your posters! 🤩

> <!-- Idea: Twitter thread with the posters in order of delivery -->

Reviewer: No hace falta incluir la idea aquí 😄 

```suggestion
```
